src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -240,31 +315,46 @@ public MateUnmapped read( final Kryo kryo, final Input input, final Class<MateUn
 
     @DefaultSerializer(InterContigPair.Serializer.class)
     public static final class InterContigPair extends BreakpointEvidence {
-        private final int mateContigIndex;
-        private final int mateStartPosition;
+        private final boolean mateReverseStrand;
+        private final SVInterval target;
 
         InterContigPair( final GATKRead read, final ReadMetadata metadata ) {
             super(read, metadata);
-            this.mateContigIndex = metadata.getContigID(read.getMateContig());
-            this.mateStartPosition = read.getMateStart();
+            final int mateContigIndex = metadata.getContigID(read.getMateContig());
+            final int mateStartPosition = read.getMateStart();
+            mateReverseStrand = read.mateIsReverseStrand();
+            final int medianFragmentSize = metadata.getStatistics(read.getReadGroup()).getMedianFragmentSize();
+            target = new SVInterval(mateContigIndex,
+                    mateReverseStrand ? mateStartPosition - medianFragmentSize : mateStartPosition + read.getLength(),
+                    mateReverseStrand ? mateStartPosition : mateStartPosition + read.getLength() + medianFragmentSize);
         }
 
         private InterContigPair( final Kryo kryo, final Input input ) {
             super(kryo, input);
-            mateContigIndex = input.readInt();
-            mateStartPosition = input.readInt();
+            mateReverseStrand = input.readBoolean();
+            target = kryo.readObject(input, SVInterval.class);
```

Comment:
It's good to avoid readObject and writeObject insofar as possible, because those methods do a complicated dance to try to maintain the integrity of the object graph.  Two alternatives to consider:  just read and write the three integers that SVInterval contains, or use the SVInterval.Serializer directly.